kong-3.0.1-0.rockspec

```diff
@@ -1,10 +1,10 @@
 package = "kong"
-version = "3.0.0-0"
+version = "3.0.1-0"
```

Comment:
I think we should be bumping to "the next version that can be released from the current master".

`release/3.0.x` is a long-lived branch. All the members of the `3.0.x` family (including `3.0.1`) will be commits on that branch.

The next release that will come out from `master` is `3.1.0`, so I think that is the correct version to "preemptively bump to".
```suggestion
version = "3.1.0-0"
```

(rockspec filename would need to be updated as well)